go-controller/pkg/util/pod_annotation.go

```diff
@@ -356,7 +358,26 @@ func SecondaryNetworkPodIPs(pod *v1.Pod, networkInfo NetInfo) ([]net.IP, error)
 	return ips, nil
 }
 
+func GetPrimaryNetworkNADNameForNamespaceFromNetInfo(namespace string, netinfo NetInfo) string {
+	for _, nadName := range netinfo.GetNADs() {
+		if !strings.HasPrefix(nadName, namespace) {
```

Comment:
`!strings.HasPrefix(nadName, namespace+'/')`

Being strict, we should be using `SplitMetaNamespaceKey` however.

Reply:
ok, using `SplitMetaNamespaceKey`  function to check namespace now.